.github/workflows/python-package.yml

```
@@ -0,0 +1,45 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+      - '**.png'
+      - 'img/*'
+      - 'LICENSE'
+      - '.gitignore'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - '**.png'
+      - 'img/*'
+      - 'LICENSE'
+      - '.gitignore'
+
+jobs:
+  build-n-test:
+
+    runs-on: ubuntu-latest
+    # See: https://github.com/marketplace/actions/skip-based-on-commit-message
+    if: "!contains(github.event.head_commit.message, 'ci skip') || !contains(github.event.head_commit.message, 'skip ci')"
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install pytest pytest-cov
+        python setup.py install
+    - name: Test with pytest
+      run: |
+        pytest --cov-report term-missing --cov-config=.coveragerc --cov=redata_commons tests
```

I believe `redata_commons` here should change to `redata`
```suggestion
        pytest --cov-report term-missing --cov-config=.coveragerc --cov=redata tests
```

---

Ah yes, I renamed and forgot to update the GitHub action.

---

@astrochun this was the minor fix I talked about in my review. Next time I'll reference things like this for clarity.